blog/_posts/2016-10-21-Team-Echo-wrapping-up.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Our end of RGSoC,:( !
+layout: post
+created_at: Thu July 28 2016
```

Comment:
please change the date to the publishing one so that it's consistent with the date in the filename